src/components/SettingsMenu/SettingsMenu.stories.tsx

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import React from 'react';
+import { List, ListItem, ListItemText } from '@material-ui/core';
+// import faker from 'faker';
+
+import Dialog from 'components/Dialog';
+import Component from './SettingsMenu';
+
+export default { title: 'SettingsMenu' };
+
+// const content = <h1>content here</h1>;
+// const content2 = (onclick: () => void) => {
+//     return (
+//         <List>
+//             <li>
+//                 <ListItem button onClick={onclick}>
+//                     <ListItemText primary='about' />
+//                 </ListItem>
+//             </li>
+//             <li>
+//                 <ListItem button>
+//                     <ListItemText primary='feedback' />
+//                 </ListItem>
+//             </li>
+//             <li>
+//                 <ListItem button>
+//                     <ListItemText primary='delete account' />
+//                 </ListItem>
+//             </li>
+//             <li>
+//                 <ListItem button>
+//                     <ListItemText primary='about the devs' />
+//                 </ListItem>
+//             </li>
+//         </List>
+//     );
+// };
+// const content3 = (
+//     <List>
+//         <li>
+//             <ListItem button>
+//                 <ListItemText primary='about' />
+//             </ListItem>
+//         </li>
+//         <li>
+//             <ListItem button>
+//                 <ListItemText primary='feedback' />
+//             </ListItem>
+//         </li>
+//         <li>
+//             <ListItem button>
+//                 <ListItemText primary='delete account' />
+//             </ListItem>
+//         </li>
+//         <li>
+//             <ListItem button>
+//                 <ListItemText primary='about the devs' />
+//             </ListItem>
+//         </li>
+//     </List>
+// );
+
+// const makeSections = (num?: number) => {
+//     const iterations = num || 1;
+//     const data = [];
+//     for (let i = 0; i < iterations; i += 1) {
+//         data.push({ title: faker.random.word(), content: content2 });
+//     }
+//     return data;
+// };
```

Comment:
If this is unnecessary, please remove.

Reply:
Commented for our own use. Reverted back to the first instance of `SettingsMenu`